modules/database/sqlite/activate.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Actions to run when the module gets activated.
+ *
+ * @since n.e.x.t
+ * @package performance-lab
+ */
+
+/**
+ * Copies the db.php file in wp-content and reloads the page.
+ *
+ * @since n.e.x.t
+ */
+return function() {
+	// Bail early if the SQLite3 class does not exist.
+	if ( ! class_exists( 'SQLite3' ) ) {
+		return;
+	}
+
+	$destination = WP_CONTENT_DIR . '/db.php';
+
+	// Bail early if the file already exists.
+	if ( file_exists( $destination ) ) {
+		return;
+	}
+
+	// Init the filesystem to allow copying the file.
+	global $wp_filesystem;
+	if ( ! $wp_filesystem ) {
+		require_once ABSPATH . '/wp-admin/includes/file.php';
+		WP_Filesystem();
+	}
+
+	// Copy the file, replacing contents as needed.
+	if ( $wp_filesystem->touch( $destination ) ) {
+
+		// Get the db.copy file contents, replace placeholders and write it to the destination.
+		$file_contents = str_replace(
+			array(
+				'{SQLITE_IMPLEMENTATION_FOLDER_PATH}',
+				'{PERFLAB_PLUGIN}',
+				'{SQLITE_MODULE}',
+				'{PERFLAB_MODULES_SETTING}',
+				'{PERFLAB_MODULES_SCREEN}',
+			),
+			array(
+				__DIR__,
+				str_replace( WP_PLUGIN_DIR . '/', '', PERFLAB_MAIN_FILE ),
+				'database/sqlite',
+				PERFLAB_MODULES_SETTING,
+				PERFLAB_MODULES_SCREEN,
```

Comment:
If the path in the plugin changes, then the check in https://github.com/WordPress/performance/blob/4badb58bd774b6bbaca8518a45d342443c286e5c/modules/database/sqlite/db.copy#L21-L24 will bail early, avoiding fatalities. That will make the site fallback to its previously used MySQL database, at which point the user can just deactivate and reactivate the module to trigger a delete & re-copy action of the `db.php` file.

This is not what we'd expect from WP-Core, but since this is a module in the performance-lab plugin, we have to make some reasonable compromises. When SQLite gets merged in Core, all files will be in their right place, and we won't even need a drop-in file. While this is in a plugin, I don't see a way to avoid things like that. It's less than perfect, but necessary.

If something changes in the plugin, there will be no fatal errors and things will keep working. The user might have to deactivate/reactivate the module, but that's a small sacrifice.

The alternative would be to use `get_file_data` on the `wp-content/db.php` and `db.copy` files, get the version, then compare them, and if the `db.php` file is older then automatically replace it. It's a perfectly fine solution and it will work just fine, but it will add a small overhead that I'd like to avoid if possible.
If and when such a change becomes necessary, then it will be easy to implement in a separate PR.